test/js/bun/io/bun-write.test.js

```diff
@@ -290,3 +290,19 @@ it.skip("Bun.write('output.html', HTMLRewriter.transform(Bun.file)))", async don
   expect(await Bun.file(outpath).text()).toBe("<div><blink>it worked!</blink></div>");
   done();
 });
+
+it("#2674", async () => {
+  const file = path.join(import.meta.dir, "big-stdout.js");
+
+  const { stderr, stdout, exitCode } = Bun.spawnSync({
+    cmd: [bunExe(), "run", file],
+    stderr: "pipe",
```

Comment:
this should use `env: bunEnv`

Reply:
added https://github.com/oven-sh/bun/pull/2850/commits/8c0f6001a3890e1397d0687c38fc3783db319c6f